compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -2213,7 +2213,7 @@ fn encode_metadata_impl(tcx: TyCtxt<'_>, path: &Path) {
     encoder.emit_raw_bytes(METADATA_HEADER);
 
     // Will be filled with the root position after encoding everything.
-    encoder.emit_raw_bytes(&[0, 0, 0, 0]);
+    encoder.emit_raw_bytes(&[0u8; 8]);
```

Comment:
Can we consistently use to_le_bytes or so? E.g., use 0u64.to_le_bytes here.

That way the 8 isn't as magical.